pyteal/compiler/optimizer/__init__.py

```diff
@@ -1 +0,0 @@
-from .optimizer import OptimizeOptions, apply_global_optimizations
```

Comment:
Please restore the optimizer package to how it was before so that our module rules aren't broken. https://github.com/algorand/pyteal/pull/247#discussion_r833510628